bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -0,0 +1,463 @@
+| nightProbabilityOfHail          | Number:Dimensionless  | Probability of Hail During The Night                    | %    | Night-time is defined as those forecast times that fall between local dusk and dawn.                                                                                                                                                                                                                                                                                                                                                    |
+| dayProbabilityOfSferics         | Number:Dimensionless  | Probability of Sferics During The Day                   | %    | This is the probability of a strike within a radius of 50km.                                                                                                                                                                                                                                                                                                                                                                            |
+| nightProbabilityOfSferics       | Number:Dimensionless  | Probability of Sferics During The Night                 | %    | This is the probability of a strike within a radius of 50km.                                                                                                                                                                                                                                                                                                                                                                            |
+
+#### Channel groups for Site API - Daily Forecast Channels:
+
+| Channel                    | Description                                       |
+|----------------------------|---------------------------------------------------|
+| currentDailyForecast       | This is the weather forecast for the current day. |
+| currentDailyForecastPlus01 | This is the weather forecast in 1 day.            |
+| currentDailyForecastPlus02 | This is the weather forecast in 2 days.           |
+| currentDailyForecastPlus03 | This is the weather forecast in 3 days.           |
+| currentDailyForecastPlus04 | This is the weather forecast in 4 days.           |
+| currentDailyForecastPlus05 | This is the weather forecast in 5 days.           |
+| currentDailyForecastPlus06 | This is the weather forecast in 6 days.           |
+
+## Full Example
+
+### Configuration (*.things)
+
+#### Site API
+
+```java
+Bridge metofficedatahub:bridge:metoffice [siteSpecificRateDailyLimit=200, siteSpecificClientId="<Site Specific Client ID>", siteSpecificClientSecret="<Site Specific Client Secret>"]{
```

Comment:
Maybe you can use an real-world id / secret that has the same format, but just with some characters manually altered?

Reply:
The new authentication uses at least for my key approx 1623 characters. That make it more unreadable otherwise would adjust the encoded contents to blat out my PII and put something representative. However given the length when this is updated I suspect it may be more readable left as is? @lsiepel

Reply:
Resolving please re-open if required @lsiepel